scripts/update_thrift_bindings.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash -eux
 
-HIVE_VERSION='2.1.1'  # Must be a released version
+HIVE_RELEASE='master' # 'rel/rel/release-2.1.1'
```

Comment:
Looks like release-2.3.0-rc1 has what you want. It's not quite a release, but I'll settle for it.